docs/Configuration.md

```diff
@@ -32,6 +32,28 @@ explanation consists of:
 
 The sections below contain links to the lists of available exclude reasons for each type of exclude.
 
+#### Excluding Paths
+
+Path excludes are used to mark a complete path as excluded. They are defined using a
+[glob pattern](https://docs.oracle.com/javase/tutorial/essential/io/fileOps.html#glob), a `reason` and a `comment`:
```

Comment:
I believe we should also explain the difference to project excludes here, or even clearly say that path excludes are a more generic replacement for project excludes, and that the latter are deprecated.

Reply:
I have added a comment about this. However, let's first discuss offline how we want to implement project specific scope excludes, before marking project excludes as deprecated.